service/history/replicatorQueueProcessor.go

```diff
@@ -555,6 +555,9 @@ func (p *replicatorQueueProcessorImpl) toReplicationTask(
 			task.SourceTaskId = common.Int64Ptr(qTask.GetTaskID())
 		}
 		return task, err
+	case persistence.ReplicationTaskTypeFailoverMarker:
+		//TODO: add function to generate failover marker task
+		return nil, nil
```

Comment:
nit: return an error?

Reply:
currently, just ignore the task or the tests will fail. I will wire this in the next PR